tests/localStorage/przykladowyranking.js

```diff
@@ -0,0 +1,88 @@
+const obj = {
+  students_medium: [
+    {
+      name: 'Test Name',
+      result: 451,
+    },
+    {
+      name: 'Test Name',
+      result: 300,
+    },
+    {
+      name: 'Test Name',
+      result: 125,
+    },
+  ],
+  students_hard: [
+    {
+      name: 'Albus Dumbledore Brother',
+      result: 451,
+    },
+    {
+      name: 'Rubeus Hagrid',
+      result: 300,
+    },
+    {
+      name: 'Severus Snape',
+      result: 125,
+    },
+  ],
+  staff_medium: [
+    {
+      name: 'Robert Mateja',
+      result: 9999,
+    },
+    {
+      name: 'Adam Małysz',
+      result: 999,
+    },
+    {
+      name: 'Narcissa Malfoy',
+      result: 3,
+    },
+  ],
+  staff_hard: [
+    {
+      name: 'Test Name',
+      result: 451,
+    },
+    {
+      name: 'Test Name',
+      result: 300,
+    },
+    {
+      name: 'Test Name',
+      result: 125,
+    },
+  ],
+  houses_medium: [
+    {
+      name: 'Gregory Goyle',
+      result: 517,
+    },
+    {
+      name: 'Test Name',
+      result: 500,
+    },
+    {
+      name: 'Test Name',
+      result: 400,
+    },
+  ],
+  houses_hard: [
```

Comment:
Great, that is exactly the kind of mock that is needed if you need to prepare UI for a functionality that is a work in progress. Good job!